library/std/src/collections/hash/map.rs

```diff
@@ -344,6 +344,11 @@ impl<K, V, S> HashMap<K, V, S> {
     ///     println!("{key}");
     /// }
     /// ```
+    ///
+    /// # Performance
+    ///
+    /// In current implementation, iterating over keys takes O(capacity) time instead
```

Comment:
Nit: should be "In *the* current implementation" here.

Reply:
Fixed :)